chalicelib_fourfront/checks/header_checks.py

```diff
@@ -314,11 +314,11 @@ def patch_static_headers_DNase_HiC(connection, **kwargs):
     return action
 
 #Chromatin Tracing
-@check_function()
+@check_function(action="patch_static_headers_Chromatin_Tracing")
```

Comment:
Why is it `patch` in the declaration and `prepare` in the definition name?

Reply:
The patch in `patch_static_headers_Chromatin_Tracing` is the name of the action function below (line 330).